test/facets/archaeologist/publishPrivateKey.spec.ts

```diff
@@ -14,7 +14,7 @@ import {
 
 const { deployments, ethers } = require("hardhat");
 
-describe("bread ArchaeologistFacet.publishPrivateKey", () => {
```

Comment:
hah I use this because I hardcode `--grep bread` into the test script in webstorm for easy debugging on specific tests. Sometimes it sneaks into draft prs.